src/components/TaskList/TaskList.jsx

```diff
@@ -219,7 +243,9 @@ class TaskList extends React.Component {
           </div>
         </nav>
         <div className="task-list-container">
-          {this.taskListOrMessage()}
+          <div className={this.state.showHidden && 'viewing-hidden'}>
```

Comment:
Yeah, I learned that is a JSX thing. If the className is falsey, it doesn't add any class.